src/style.css

```diff
@@ -0,0 +1,84 @@
+html {
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.form {
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+.form:hover {
+  box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.input1::placeholder {
+  font-size: 16px;
+}
+
+.input2::placeholder {
+  font-size: 14px;
+}
+
+.input {
+  width: 100%;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  border: none;
+  color: #3d4e61;
+  display: inline-block;
+}
+
+.input1 {
+  height: 68px;
```

Comment:
I guess, it has 70px height.

Use border-box, and you don't need to calcute difference between borders